src/qrl/core/State.py

```diff
@@ -26,6 +26,12 @@
 
 
 class State:
+    """
+    State is the only class that is allowed to read/write to the database.
+    Everything else should go through it. Basically State manages the database
+    and provides a business logic interface that other components can use.
```

Comment:
1. "Everything else should go through it." <-- This doesnt make sense. Everything doesnt need to go through it. There are many classes, which doesnt need State.

Reply:
I meant 'needs to go through it to read/write to the database'. Deleted the sentence.